network/manager.go

```diff
@@ -192,13 +193,19 @@ func (nm *networkManager) restore(isRehydrationRequired bool) error {
 	// Read any persisted state.
 	err := nm.store.Read(storeKey, nm)
 	if err != nil {
```

Comment:
Does it make sense to log the contents invalid and all to help assist with troubleshooting?

Reply:
I was looking into this and the store's file name and contents are not accessible-- currently leaning towards just adding a method to the interface to dump the contents of the store in string format.